attendance.py

```diff
@@ -38,8 +39,11 @@ async def main():
         os.makedirs("./exports")
 
     for e in events:
+        filename = f"{e['startTimestamp']}-{e['heading']}.csv"
```

Comment:
Would be nice to refcator this out into self-documenting function, e.g. `santised_filename(event)`